docs/customization/translation.rst

```diff
@@ -41,6 +46,16 @@ override its translation key which is ``sylius.form.customer.email``.
             customer:
                 email: Username
 
+Before changes
+
+.. image:: ../_images/before_customizing_translation.png
+    :align: center
+
+After changes
```

Comment:
To be consistent with https://github.com/Sylius/Sylius/pull/10353/ I would say it should be just `Before:` and `After:`